### PR TITLE
docs: rewrite intro page

### PR DESCRIPTION
**Pull request by meaghanfitzgerald**
This PR contains a re-write of the Intro page. 

Current state: https://docs.avax.network/intro

### CODE REVIEW

**Review by dubrado — APPROVED** [review]

**Review by usmaneth — DISMISSED** [review]

File: docs/learn/avalanche/intro.md [hunk 1]
```diff
@@ -3,55 +3,39 @@ slug: /intro
...
 ---
 
-# Introduction
+# What Is Avalanche?
 
-Avalanche is an open-source platform for building decentralized applications (Dapps) in one
-interoperable, highly scalable ecosystem. Avalanche is the first ecosystem designed to accommodate
-the scale of global finance, with near-instant transaction finality.
+Avalanche is an open-source platform for building decentralized applications in one
+interoperable, decentralized, and highly scalable ecosystem. Powered by a uniquely powerful consensus
```
Comment (cgcardona) [inline docs/learn/avalanche/intro.md:9]: Change the following string:

`Powered by a uniquely powerful consensus mechanism`

to the following link, which points to the Avalanche Consensus tutorial.

`Powered by [a uniquely powerful consensus mechanism](./learn/avalanche/avalanche-consensus),`

File: docs/learn/avalanche/intro.md [hunk 2]
```diff
@@ -3,55 +3,39 @@ slug: /intro
...
-Bitcoin.
+- Avalanche employs the fastest consensus mechanism of any layer 1 blockchain. The unique consensus
+mechanism enables quick finality and low latency: in less than 2 seconds, your transaction is
+effectively processed and verified.
 
-## Building on Avalanche
+### Built to Scale
 
-### Virtual Machines
+- Anywhere from 1,000 to 10,000 transactions are confirmed per second, virtually unaffected by network
```
Comment (cgcardona) [inline docs/learn/avalanche/intro.md:23]: > Anywhere from 1,000 to 10,000 transactions are confirmed per second

On [the 5-23-23 Dev Community call, @ 21:29](https://www.youtube.com/live/HD3vdZzraE8?feature=share&t=1289), POG announced 50K TPS

**Review by cgcardona — CHANGES_REQUESTED** [review]
Looks great! I left a couple of comments requesting minor changes. 

1. Link to the Consensus tutorial when mentioning Consensus.
2. Update the upper-bound of the TPS count and optionally link out to the timestamp when POG mentions it in yesterday's Developer Community Call.

File: docs/learn/avalanche/intro.md (hunk 2 above)
Comment (meaghanfitzgerald) [reply docs/learn/avalanche/intro.md:23]: Patrick was speaking specifically about an instance where he tested a hyperSDK implementation. I've talked with Stephen and we decided to remove this statement all together.

**Review by dubrado — APPROVED** [review]

**Pull request merged**